ui/src/components/dialogs/ManageUsersDialog.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+    <div v-if="shareSessionButtonIsVisible">
+        <div style="padding: 0 10px">
+            <el-badge :value="allowedUserProfiles.length === 0 ? undefined : allowedUserProfiles.length" style="width: 135px">
+                <el-button @click="dialogIsVisible = true" class="share-session-button" type="primary">
+                    Share session
+                </el-button>
+            </el-badge>
+        </div>
+        <el-dialog :append-to-body="true" title="Share session with other users" v-model="dialogIsVisible" :before-close="handleClose">
+            <div style="width: 100%">
+                <h3 style="margin-bottom: 0;">Sharing with</h3>
+                <p style="color: rgb(167, 167, 167)">The following users have access to this session</p>
+                <div>
+                    <p style="color: rgb(217 190 46)" v-if="allowedUserProfiles.length === 0">No users added</p>
+                    <div v-else>
+                        <el-tag
+                            style="margin: 4px"
+                            size="large"
+                            closable
+                            v-for="allowedUser in allowedUserProfiles"
+                            :key="allowedUser?.sub"
```

Comment:
for some reason, github highlighting breaks at this point, I think it does not understand the `?.` yet

Reply:
Based on my testing, this should be fine